Assets/PlayerInput.cs

```diff
@@ -181,19 +227,31 @@ public void AddCallbacks(IMouseKeyboardActions instance)
             @Move.started += instance.OnMove;
             @Move.performed += instance.OnMove;
             @Move.canceled += instance.OnMove;
-            @Plant.started += instance.OnPlant;
-            @Plant.performed += instance.OnPlant;
-            @Plant.canceled += instance.OnPlant;
+            @Plant1.started += instance.OnPlant1;
+            @Plant1.performed += instance.OnPlant1;
+            @Plant1.canceled += instance.OnPlant1;
+            @Plant2.started += instance.OnPlant2;
+            @Plant2.performed += instance.OnPlant2;
+            @Plant2.canceled += instance.OnPlant2;
+            @Plant3.started += instance.OnPlant3;
+            @Plant3.performed += instance.OnPlant3;
+            @Plant3.canceled += instance.OnPlant3;
         }
 
         private void UnregisterCallbacks(IMouseKeyboardActions instance)
         {
             @Move.started -= instance.OnMove;
             @Move.performed -= instance.OnMove;
             @Move.canceled -= instance.OnMove;
-            @Plant.started -= instance.OnPlant;
```

Comment:
Whats good practice is to use a for each loop and a list to contain all these to make it cleaner